system/Validation/Exceptions/ValidationException.php

```diff
@@ -39,4 +39,27 @@ public static function forNoRuleSets()
 	{
 		return new static(lang('Validation.noRuleSets'));
 	}
+
+	public static function forInvalidRuleCollectionMissingPropertyRules($rule = null) : self
+	{
+		return new static(lang('Validation.invalidRuleCollection', [ static::convertRuleToString($rule), lang('Validation.missingPropertyRules') ]));
+	}
+
+	/**
+	 * Used as a helper to convert anything else than a string to a rule (name)
+	 *
+	 * @param  mixed $rule
+	 * @return string|null
+	 */
+	protected static function convertRuleToString($rule) : ?string
+	{
+		if (! is_string($rule))
+		{
+			// Try to not confuse the developer with something like "Cannot use array as a rule."
+			// but don't leak to much
+			$rule = ENVIRONMENT !== 'production' ? '"' . var_export($rule, true) . '"' : gettype($rule);
```

Comment:
Why is double quote needed on `var_export`?

Reply:
it's not indeed needed, just for readability.
Moved this to the translation.